music21/instrument.py

```diff
@@ -2167,24 +2168,22 @@ def partitionByInstrument(streamObj):
     TODO: use proper recursion to make a copy of the stream.
     TODO: final barlines should be aligned.
     '''
-    from music21 import stream
```

Comment:
moved this to the top of the module.

Reply:
ah!  This used to have recursion problems.  Probably fixed with the base/core separation!